src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,23 @@
+import { jwtService } from '../services/jwt.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  const userData = jwtService.verifyToken(token);
```

Comment:
This line assumes that jwtService.verifyToken will return falsy if the token is invalid. Ensure that verifyToken is implemented to return null or undefined for invalid tokens, or handle exceptions if it throws them.